packages/tabs/src/types.ts

```diff
@@ -6,9 +6,11 @@
  */
 
 import { IUseSelectionProps, IUseSelectionReturnValue } from '@zendeskgarden/container-selection';
-import { HTMLProps, ReactNode, RefObject } from 'react';
+import { HTMLProps, MutableRefObject, ReactNode } from 'react';
 
 export interface IUseTabsProps<Item> extends Omit<IUseSelectionProps<Item>, 'direction'> {
+  /** The array of IDs used for managing selection */
```

Comment:
```suggestion
  /** Provides an ordered list of unique tab values */
```

Doc grammar. Compare https://github.com/zendeskgarden/react-containers/blob/3676836c2463bee4163139299f156bffa7d26a39/packages/combobox/src/types.ts#L45-L47